src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,7 +577,23 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
-    
+
+    protected static List<FeedbackParticipantType> getParticipantListFromParams(String participantListParam) {
```

Comment:
I've looked through the method. It seems like we don't need a new abstract action class, but rather a new method in `FeedbackParticipantType`.